{"sections":[],"hierarchy":{"paths":[["doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError"]]},"variants":[{"paths":["\/documentation\/mdots\/autherror\/unknown"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"role":"symbol","modules":[{"name":"MDots"}],"externalID":"s:5MDots9AuthErrorO7unknownyA2CmF","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"unknown"}],"symbolKind":"case","title":"AuthError.unknown","roleHeading":"Case"},"kind":"symbol","identifier":{"url":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/unknown","interfaceLanguage":"swift"},"schemaVersion":{"patch":0,"major":0,"minor":3},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"unknown","kind":"identifier"}]}]}],"references":{"doc://com.estelapasjak.Mdots/documentation/MDots":{"url":"\/documentation\/mdots","kind":"symbol","abstract":[],"role":"collection","images":[{"type":"icon","identifier":"new-icon.png"}],"identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots","type":"topic","title":"MDots"},"new-icon.png":{"type":"image","variants":[{"traits":["1x","light"],"url":"\/images\/new-icon.png"}],"identifier":"new-icon.png","alt":"The logo representing the MDot sensor application."},"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError":{"kind":"symbol","navigatorTitle":[{"text":"AuthError","kind":"identifier"}],"title":"AuthError","type":"topic","abstract":[{"text":"An enumeration representing various authentication errors that can occur.","type":"text"}],"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"AuthError","kind":"identifier"}],"url":"\/documentation\/mdots\/autherror","identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError"},"doc://com.estelapasjak.Mdots/documentation/MDots/AuthError/unknown":{"url":"\/documentation\/mdots\/autherror\/unknown","kind":"symbol","abstract":[],"role":"symbol","identifier":"doc:\/\/com.estelapasjak.Mdots\/documentation\/MDots\/AuthError\/unknown","type":"topic","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"unknown","kind":"identifier"}],"title":"AuthError.unknown"}}}